.eleventy.js

```diff
@@ -4,13 +4,15 @@ const syntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
 const pluginRss = require("@11ty/eleventy-plugin-rss");
 
 module.exports = function(eleventyConfig) {
-  eleventyConfig.setTemplateFormats(["md", "njk", "html", "css", "pdf"]);
+  eleventyConfig.setTemplateFormats(["md", "njk", "html", "css", "pdf", "liquid"]);
 
   eleventyConfig.addPassthroughCopy("CNAME");
   eleventyConfig.addPassthroughCopy({"assets/img": "img"});
   eleventyConfig.addPassthroughCopy({"assets/svg": "svg"});
-  eleventyConfig.addPassthroughCopy({"node_modules/reveal.js/dist": "reveal.js/dist"});
-  eleventyConfig.addPassthroughCopy({"node_modules/reveal.js/plugin": "reveal.js/plugin"});
+eleventyConfig.addPassthroughCopy({"node_modules/reveal.js/dist/js": "reveal.js/dist/js"});
```

Comment:
Nit: intendation doesn't match the rest of the code.

Reply:
Fixed in 38c1eb978e738c2d6ff896abea824a8f13703229